components/raftstore-v2/src/operation/command/admin/mod.rs

```diff
@@ -96,6 +96,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             return;
         }
 
+        let is_transfer_leader = cmd_type == AdminCmdType::TransferLeader;
```

Comment:
why not use the `pre_transfer_leader` the variable.

Reply:
`pre_transfer_leader` has some requirements to request flag.